server/routes/courtRegister/courtRegisterController.ts

```diff
@@ -125,4 +126,15 @@ export default class CourtRegisterController {
     const { id, name } = req.session.addNewCourtForm
     res.render('pages/court-register/addNewCourtFinished', { id, name })
   }
+
+  async amendCourtDetails(req: Request, res: Response): Promise<void> {
+    const { courtId } = req.query as { courtId: string }
+
+    const court = await this.courtRegisterService.getCourt(context(res), courtId)
```

Comment:
seems a bit naff to call these in turn, could this be rewritten to
```javascript
const [court, courtTypes] = await Promise.all([
    this.courtRegisterService.getCourt(context(res), courtId),
    this.courtRegisterService.getCourtTypes(context(res))
])
```

Reply:
yeah, I thought about that - I make them run in a parallel- good call